workbench/src/main/setupAddRemovePlugin.js

```diff
@@ -151,11 +176,13 @@ export function setupAddPlugin(i18n) {
             modelTitle: pluginTitle,
             type: 'plugin',
             source: installString,
-            env: pluginEnvPrefix,
+            env: newPluginEnvPrefix,
+            version: version,
           }
         );
         logger.info('successfully added plugin');
       } catch (error) {
+        console.log(error);
```

Comment:
Leave in or for debugging?

Reply:
I think this will be useful in the future for debugging